edb/ir/typeutils.py

```diff
@@ -101,6 +101,18 @@ def is_abstract(typeref: irast.TypeRef) -> bool:
     return typeref.is_abstract
 
 
+def is_persistent_tuple(typeref: irast.TypeRef) -> bool:
```

Comment:
Why not `is_material_tuple`? What's the difference between "material" and "persistent"?

Reply:
"material" means non-derived, i.e. not an ephemeral subtype derived for compilation purposes. "persistent" means that the type is used in schema objects (other than transiently in some expression), and will be loaded with schema introspection.

Reply:
Is there a place you can document this and alike terminology?

Reply:
Yes. I'll make a pass and add some docs.